Makefile.inc

```diff
@@ -9,4 +9,9 @@ MAKE      = make
 COPT      = -O3
 CWARN     = -Wno-long-long
 CFLAGS    = -MD -MP -std=c11 -Wall -Wpedantic -Wextra $(CWARN) $(COPT) $(CINC) -Wno-unused-function
+
+ifeq ($(ARCHNAME), x86)
+CFLAGS   += -m32
```

Comment:
Is there not an equivalent flag for x86-64?

Reply:
Should we have more options/conditions here?

Or should we populate the `CFLAGS` starting with `configure`.  That seems the cleanest.

Reply:
Yes, -m64 is for x86-64. We should make the configure script handle that, you're right.

I agree that configure setting up all the configuration based CFLAGS in Makefile.config it creates with variables for different configurations.